### PR TITLE
Structure, logic and structure

### PR DESCRIPTION
**Pull request by gdumani**
**This pull request implements a React/Redux app with the following features:**
- Access to the Gutendex API to the Gutenberg Open Library.
- Display a collection of books in 32 item pages.
-  Page navigation
- Collection is created using topic, author/title, and language filters.
- Books from a page can be selected and details will be displayed.
- Books can be retrieved to be read on the screen.
Axios is used to interface with the API
A test suite for the components and reducers is included.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,24 +1,72 @@
-.App {
+.main {
+  background-color: #ff5698;
+}
```
Comment (levy002) [inline src/App.css]: - [x] Kindly make sure your background image fits the whole size of the screen in order to match the Figma design as much as possible.

   _**Hint: You can add the `height` property with `100vh` as a value which will give the `main` container ability to fit the whole visible screen**_


   ### YOUR DESIGN
    
   image


   ### YOUR DESIGN AFTER ADDING THE HEIGHT
     image


   ### FIGMA DESIGN
    image

File: src/App.css [hunk 2]
```diff
@@ -1,24 +1,72 @@
...
+
+img {
+  margin-left: 1rem;
+}
+
+.back {
+  max-width: 25px;
+  height: auto;
 }
```
Comment (levy002) [inline src/App.css]: - [x] Kindly make sure to align your `back` button correctly in order to match the provided design.


    - ### YOUR DESIGN
    
     image

    - ### PROVIDED DESIGN
    
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @gdumani   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Well-detailed README :heavy_check_mark: 
- Your app is functioning well as expected :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @gdumani, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
- Made Requested Changes By Previous Reviewer :heavy_check_mark: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**